uniswap-v2/src/common/traits.rs

```diff
@@ -0,0 +1,30 @@
+use ethabi::ethereum_types::Address;
+use substreams::store::{StoreGet, StoreGetProto};
+use substreams_helper::common::HasAddresser;
+use substreams_helper::hex::Hexable;
+
+use crate::pb::uniswap::v2::Pool;
+use crate::store_key::StoreKey;
+
+pub struct PoolAddresser<'a> {
+    pub store: &'a StoreGetProto<Pool>,
```

Comment:
HasAddresser is [already implemented for all stores](https://github.com/messari/substreams/blob/master/substreams-helper/src/common.rs#L35), including `StoreGetProto`, unless I did something wrong.

You can pass the store directly as a HasAddresser without implementing the trait again

Reply:
The key for the store is set to the address of the event. whereas I need to pass my own [key](https://github.com/messari/substreams/blob/39a889a296d35d2e0347446045778fe1e5e52631/uniswap-v2/src/common/traits.rs#L15-L17) to verify if the address is available in store or not.